### PR TITLE
fix: English locale

### PR DESCRIPTION
**Pull request by NaMax66**
Description:
https://virtocommerce.atlassian.net/browse/ST-4865

--

QA-test:

Demo-test:

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-1.38.0-pr-699-ca35-ca35cadf.zip

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -1000,17 +1000,17 @@
         "billing_address": "Billing Address",
         "save_changes": "@:common.buttons.save_changes",
         "submit": "@:common.buttons.submit",
-        "no_address_message": "You do not set an address. Please specify one.",
+        "no_address_message": "You have not set an address. Please specify one.",
```
Comment (asvishnyakov) [inline locales/en.json:1003]: `You didn't` and I think it's better `You should provide an address` or something like that.

File: locales/en.json [hunk 2]
```diff
@@ -1180,8 +1180,8 @@
     "bulk_order": {
       "meta": {
         "title": "Bulk order",
-        "keywords": "test, balk, order, keywords",
-        "description": "Test balk order page description"
+        "keywords": "test, bulk, order, keywords",
+        "description": "Test bulk order page description"
```
Comment (asvishnyakov) [inline locales/en.json:1184]: What's this `test` messages?

File: locales/en.json [hunk 3]
```diff
@@ -826,9 +826,9 @@
       "text": "Please use your username and password to login to your account.",
       "continue_button": "@:shared.layout.header.link_sign_in",
       "continue_shopping_button": "@:common.buttons.continue_shopping",
-      "email_confirmation_failed_text": "In order to continue you still need to finalize verification process. Please use the button below to request a new link.",
-      "link_sent_successfully_text": "We sent you a email confirmation link to your email.",
-      "link_sent_error_text": "Some problems are occured while sending a comfirmation link. Please try again later."
+      "email_confirmation_failed_text": "In order to continue, you still need to finalize verification process. Please use the button below to request a new link.",
+      "link_sent_successfully_text": "We sent you the confirmation link to your email.",
+      "link_sent_error_text": "Some problems have occurred while sending the confirmation link. Please try again later."
```
Comment (asvishnyakov) [inline locales/en.json:831]: Just `occured` or even `A problem occured`

**Review by asvishnyakov — CHANGES_REQUESTED** [review]

File: locales/en.json (hunk 2 above)
Comment (NaMax66) [reply locales/en.json:1184]: image

Comment (asvishnyakov) [reply locales/en.json:1184]: I think we need to discuss about that with @aavolko and replace this test messages to something production-ready

File: locales/en.json (hunk 1 above)
Comment (NaMax66) [reply locales/en.json:1003]: Discussed @aavolko keep You have not set an address. Please specify one.

File: locales/en.json (hunk 2 above)
Comment (NaMax66) [reply locales/en.json:1184]: @aavolko remove test

File: locales/en.json (hunk 3 above)
Comment (NaMax66) [reply locales/en.json:831]: @aavolko  Some problems have occurred -> A problem has occurred

File: locales/en.json [hunk 4]
```diff
@@ -458,17 +458,17 @@
...
       "add_to_cart": {
         "errors": {
           "in_cart": "in Cart",
-          "already_in_cart_message": "{0} has already been added to cart",
-          "enter_correct_number_message": "Please enter valid number",
-          "min": "Min qty is {0} product(s)",
-          "max": "Max qty is {0} product(s)"
+          "already_in_cart_message": "{0} has already been added to the cart",
+          "enter_correct_number_message": "Please enter a valid number",
+          "min": "Minimum quantity is {0} product(s)",
```
Comment (NaMax66) [inline locales/en.json:470]: The

File: locales/en.json [hunk 5]
```diff
@@ -565,11 +565,11 @@
           "same_as_shipping_address": "Same as shipping address"
         },
         "links": {
-          "select_address": "select a billing address"
+          "select_address": "Select a billing address"
         },
         "messages": {
-          "no_addresses": "You do not yet have any billing address. Please choose or specify one.",
-          "unauthenticated_no_addresses": "You do not yet have any billing address. Please specify one."
+          "no_addresses": "You do not yet have any billing addresses. Please choose or specify one.",
```
Comment (NaMax66) [inline locales/en.json:571]: - yet

File: locales/en.json [hunk 6]
```diff
@@ -565,11 +565,11 @@
...
+          "unauthenticated_no_addresses": "You do not yet have any billing addresses. Please specify one."
```
Comment (NaMax66) [inline locales/en.json:572]: minus yet

File: locales/en.json [hunk 7]
```diff
@@ -826,9 +826,9 @@
       "text": "Please use your username and password to login to your account.",
       "continue_button": "@:shared.layout.header.link_sign_in",
       "continue_shopping_button": "@:common.buttons.continue_shopping",
-      "email_confirmation_failed_text": "In order to continue you still need to finalize verification process. Please use the button below to request a new link.",
-      "link_sent_successfully_text": "We sent you a email confirmation link to your email.",
-      "link_sent_error_text": "Some problems are occured while sending a comfirmation link. Please try again later."
+      "email_confirmation_failed_text": "In order to continue, you still need to finalize verification process. Please use the button below to request a new link.",
+      "link_sent_successfully_text": "We sent you the confirmation link to your email.",
```
Comment (NaMax66) [inline locales/en.json:830]: ...to your inbox

File: locales/en.json [hunk 8]
```diff
@@ -1280,26 +1280,26 @@
...
     "PasswordMismatch": "You entered the wrong current password, please try again or call support",
-    "PasswordRequiresLower": "Passwords must have at least one lowercase ('a'-'z')",
-    "PasswordRequiresNonAlphanumeric": "Passwords must have at least one non alphanumeric character",
-    "PasswordRequiresUpper": "Passwords must have at least one uppercase ('A'-'Z')",
+    "PasswordRequiresLower": "Passwords must have at least one lowercase letter ('a'-'z')",
+    "PasswordRequiresNonAlphanumeric": "Passwords must have at least one non-alphanumeric character",
+    "PasswordRequiresUpper": "Passwords must have at least one uppercase letter ('A'-'Z')",
     "PasswordRequiresUniqueChars": "Passwords must use different characters",
-    "RecentPasswordUsed": "You have used this password in the past. Choose another one."
+    "RecentPasswordUsed": "You have used this password in the past. Please choose another one."
```
Comment (NaMax66) [inline locales/en.json:1302]: Please select

Comment (NaMax66) [reply locales/en.json]: @aavolko or please create?

**Pull request merged**